pkgs/tools/nix/alejandra/default.nix

```diff
@@ -0,0 +1,25 @@
+{ lib
+, rustPlatform
+, fetchFromGitHub
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "alejandra";
+  version = "unstable-2022-01-30";
+
+  src = fetchFromGitHub {
+    owner = "kamadorueda";
+    repo = "alejandra";
+    rev = "b72274b052ae06cbe60a97d623829b1458369cc2";
+    sha256 = "sha256-snq9C/a/53VivFcLNvdeKwVmPBXbcVzbbjTB+iULFUc=";
+  };
+
+  cargoSha256 = "sha256-/JzATzRhNexmyjtgjHVkw8LVyr4PdIdPJfUGXz4pZbQ=";
```

Comment:
Here it could be more maintainable to use the same cargo.lock that ${src} contains

like this: `cargoLock.lockFile = "${src}/Cargo.lock"`, this way you don't have to enter a hash manually

https://github.com/kamadorueda/alejandra/blob/c07a964d8ae8d283f4c270bd26bbd391d274f729/flake.nix#L42